src/api/bucket_api.js

```diff
@@ -786,7 +786,16 @@ module.exports = {
                     }
                 },
                 num_objects: {
-                    type: 'integer'
+                    type: 'object',
```

Comment:
@nb-ohad did you verify that no other callers in core/operator are relying on that type?
In general we should prefer to add fields rather than changing existing ones.

Reply:
@guymguym I did check the core, did not check the operator, will do now.
As a general note, I agree with your last statement but the last update is so tightened to the num_object field that I feel it will be a bad habit (and a bad API structure) to split them apart.

Reply:
There is no use of this specific API output in the operator